### PR TITLE
refactor(payments): payment form tailwind convert

### PR DESCRIPTION
**Pull request by StaberindeZA**
## Because

- We want to convert the PaymentForm to using Tailwind classes

## This pull request

- Replaces all existing classes with Tailwind classes
- Updates fxa-react LoadingSpinner component to support multiple images

## Issue that this pull request solves

Closes: #fxa-5633

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [x] I have added necessary documentation (if appropriate).
- [x] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)

Please attach the screenshots of the changes made in case of change in user interface.

## Other information (Optional)

Any other information that is important to this pull request.

### CODE REVIEW

File: packages/fxa-payments-server/src/components/PaymentForm/index.tsx [hunk 1]
```diff
@@ -194,17 +197,17 @@ export const PaymentForm = ({
...
+      <div className="flex text-[15px] justify-between">
         <Localized
           id={`plan-price-${plan.interval}`}
           vars={{
             amount: getLocalizedCurrency(plan.amount, plan.currency),
             intervalCount: plan.interval_count!,
           }}
         >
-          <div className="pricing"></div>
+          <div className="flex items-center"></div>
```
[the remark this answers is not in the document]
Comment (StaberindeZA) [reply packages/fxa-payments-server/src/components/PaymentForm/index.tsx:208]: Wow, that's what you get for just blind editing something. Got rid of this entire section. Seems to be legacy.

File: packages/fxa-payments-server/src/components/PaymentForm/index.tsx [hunk 2]
```diff
@@ -194,17 +197,17 @@ export const PaymentForm = ({
...
         </Localized>
-        <PaymentProviderDetails customer={customer!} />
+        <PaymentProviderDetails customer={customer!} cardClassName="m-0" />
```
[the remark this answers is not in the document]
Comment (StaberindeZA) [reply packages/fxa-payments-server/src/components/PaymentForm/index.tsx:210]: Ah nice spot. This doesn't seem to impact other places it gets used either. Made the change.

File: packages/fxa-react/styles/animations.css [hunk 3]
```diff
@@ -4,10 +4,9 @@
 
 .transition-standard {
   @apply transition duration-150;
-
-  &:active {
-    @apply transition duration-150;
-  }
+}
+.transition-standard:active {
```
[the remark this answers is not in the document]
Comment (StaberindeZA) [reply packages/fxa-react/styles/animations.css:8]: Lol yeah I was trying to indicate that these are all the same class still. Doesn't look great though.

File: packages/fxa-payments-server/src/components/PaymentForm/index.tsx [hunk 4]
```diff
@@ -291,7 +298,9 @@ export const PaymentForm = ({
                   : payButtonL10nId(customer)
               }
             >
-              <span className="lock">{submitButtonCopy}</span>
+              <span className="before:bg-[url('../components/PaymentForm/images/lock.svg')] before:bg-[0_2px] before:bg-no-repeat before:content-['\00a0'] before:inline-block before:h-[18px] before:my-0 before:mx-2 before:w-3.5">
```
[the remark this answers is not in the document]
Comment (StaberindeZA) [reply packages/fxa-payments-server/src/components/PaymentForm/index.tsx:301]: Thanks for sharing the code. That was super useful!

File: packages/fxa-payments-server/public/locales/en-US/main.ftl [hunk 5]
```diff
@@ -29,6 +29,9 @@ brand-name-firefox-logo = { -brand-name-firefox } logo
 close-aria =
   .aria-label = Close modal
 
+# Aria label for spinner image indicating data is loading
+app-loading-spinner-aria-label-loading = Loading...
```
Comment (bcolsson) [inline packages/fxa-payments-server/public/locales/en-US/main.ftl:33]: Not an issue on localization side, but pointing out this isn't defined as an attribute compared to other .aria-label strings.

Comment (vpomerleau) [reply packages/fxa-payments-server/public/locales/en-US/main.ftl:33]: Note: This ID/message will be moved to fxa-react in issue #FXA-5998.

An alternative, from [draft PR-14440](https://github.com/mozilla/fxa/pull/14440):

```
# Aria label for loading spinner
app-loading-spinner-aria =
  .aria-label = Loading...
```

in component:
```
      <Localized id="app-loading-spinner-aria">
        <Spinner
          className="w-10 h-10 animate-spin"
          role="img"
          aria-label="Loading..."
        />
      </Localized>
```

**Pull request merged**